src/seedsigner/helpers/bip39/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .english_list import WORDLIST__ENGLISH
+from .spanish_list import WORDLIST__SPANISH
+from .french_list import WORDLIST__FRENCH
+from .italian_list import WORDLIST__ITALIAN
+from .portuguese_list import WORDLIST__PORTUGUESE
```

Comment:
It feels a bit bloat-y to import all the lists when in most cases users will need NONE of them or at most ONE of them.

I haven't looked through the whole PR so maybe this is already mostly included, but I think it might be better to have some "get_wordlist()" utility that only does the `import` within an `if`/`elif`.

I don't think importing all of them would have any impact on load times or RAM usage since SeedSigner OS loads everything into RAM at boot time anyway. But for the eventual microcontroller port (e.g. esp32-s3, rp2350), we definitely need to consider the much more limited RAM environment (likely 8MB).